src/main/java/spireMapOverhaul/zones/heavenlyClouds/HeavenlyCloudsZone.java

```diff
@@ -0,0 +1,60 @@
+package spireMapOverhaul.zones.heavenlyClouds;
+
+import basemod.helpers.CardModifierManager;
+import com.badlogic.gdx.graphics.Color;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import spireMapOverhaul.abstracts.AbstractZone;
+import spireMapOverhaul.util.Wiz;
+import spireMapOverhaul.zoneInterfaces.CombatModifyingZone;
+import spireMapOverhaul.zoneInterfaces.RewardModifyingZone;
+
+import java.util.ArrayList;
+
+import static spireMapOverhaul.util.Wiz.atb;
+
+public class HeavenlyCloudsZone extends AbstractZone implements CombatModifyingZone, RewardModifyingZone {
+    public static final String ID = "HeavenlyClouds";
+    private static final float HEALTH_MODIFIER = 0.70f;
+    public HeavenlyCloudsZone() {
+        super(ID, Icons.MONSTER);
+        this.width = 3;
+        this.maxHeight = 4;
+        this.height = 4;
+        this.maxHeight = 5;
+    }
+
+    @Override
+    public AbstractZone copy() {
+        return new HeavenlyCloudsZone();
+    }
+
+    @Override
+    public Color getColor() {
+        return Color.SKY.cpy();
+    }
+
+    @Override
+    public void atBattleStart() {
+        int flightAmount = 3;
```

Comment:
Cool zone! Flight is a good combat modifier for giving things a twist. Players can even be strategic about entering or avoiding this zone based on how well their deck does against Flight.

For balance, I think it's important to have the amount of Flight given to enemies vary. The Byrds fight is challenging because its at the start of act 2 and has three enemies, so even if you manage enough attacks in a turn there are still two other Byrds hitting you. The further away from that you get, the less balanced these specific values are.

With this static amount of flight, I would expect single enemy fights to be substantially easier, and act 3 fights to also be substantially easier. Think about fighting Gremlin Nob or Lagavulin at low ascension, and being able to stun them with 3 attacks. That's something you can do in act 1 no problem. It's perfectly fine for this zone to affect fights unequally, but some tuning of these values would go a long way towards making the zone feel fair.

There's a variety of ways to address this, but the most straightforward in my mind is to have the amount of Flight vary by act, whether the enemy is an elite, and whether the fight has only one enemy. Here's what I'd propose:
* All single enemy fights get +1 (otherwise they're just too easy)
* All elites in act 2 and 3 get +1 (because elites should stay challenging)
* All enemies in act 3 get +1 (since by that point it's a lot easier to play many attacks in a turn)

Lots of other ways to set this up, but that's my first stab at values for this that I think will be reasonably balanced.

Reply:
Act 3 elites getting 5 flight sounds terrifying that has to be tested. Playing that many attacks is hard for many decks and realsitically could lead to a lot of deaths and making this zone an instant avoid

Reply:
I definitely agree that single enemy encounters should get +1 flight; I'm not so sure about the other two additions since I agree with Gk that having these effects potentially stack up to 5 or more Flight would likely be problematic. Unfortunately the mechanic is rather volatile to balance since the addition of +/-1 Flight could be the difference between wiping to an encounter or beating it with ease 😛 

I think in this instance, I would rather make encounters too easy instead of too hard since that would likely provide a better experience for the player on average.

Reply:
I'm still concerned that the fights will be too easy too often, but we can give it a try and see if it needs adjustment. Act 1 and 2 at least won't be too far off balance with low numbers.

Though could we at least make this zone not spawn in act 3? If there's no adjustment to the flight numbers to account for having an additional boss relic and a much stronger deck, this zone will be easier fights plus better rewards for a lot of decks.

Reply:
Having it not spawn in act 3 is fair enough, I suppose

Reply:
Encounters with only a single non-minion enemy now get +1 Flight. The zone also no longer spawns in act 3.